app/draft_utils.py

```diff
@@ -17,45 +16,11 @@ def get_request_page_questions():
     return json_payload.get('page_questions', [])
 
 
-def get_draft_validation_errors(draft_json, framework_id=0, slug=None, required=None):
-    if not slug and not framework_id:
-        raise Exception('Validation requires either framework_id or slug')
-    if not slug:
-        framework = Framework.query.filter(
-            Framework.id == framework_id
-        ).first()
-        slug = framework.slug
-
+def get_draft_validation_errors(draft_json, framework=None, lot=None, required=None):
     errs = get_validation_errors(
-        "services-{0}-{1}".format(slug, draft_json['lot'].lower()),
+        "services-{0}-{1}".format(framework.slug, lot.slug),
```

Comment:
Seems like this method requires a framework and lot (`.slug` will blow up otherwise), so there's no point in `=None` as defaults in the signature?